modules/ai/assets/js/gutenberg/featured-image.js

```diff
@@ -0,0 +1,46 @@
+import { RequestIdsProvider } from '../editor/context/requests-ids';
+import { __ } from '@wordpress/i18n';
+import { AIMediaGenerateApp } from '../media-library/componenets';
+import { useGutenbergPostText } from './post-text-utils';
+import { AiLink, Icon } from './styles';
+import { useState } from '@wordpress/element';
+import useFeaturedImagePrompt from '../editor/hooks/use-featured-image-prompt';
+
+const { useDispatch } = wp.data;
+
+const GenerateFeaturedImageWithAI = () => {
+	const FEATURED_IMAGE_RATIO = '4:3';
```

Comment:
why do we need to override the default 1:1 ratio?

Reply:
Featured Image shouldn't be square and not a rectangle